src/python/grpcio/grpc/__init__.py

```diff
@@ -424,6 +424,23 @@ def __init__(self, credentials):
         self._credentials = credentials
 
 
+class ServerCertificateConfig(object):
+    """A certificate config for use with an SSL-enabled Server, e.g., can
+    be returned in the certificate config fetching callback.
+
+    This class has no supported interface -- it exists to define the
+    type of its instances and its instances exist to be passed to
+    other functions.
+
+    NOTE: do not re-use instances of this class, i.e, always create a
+    new instance when passing/returning to the library.
+
+    """
+
```

Comment:
Within the gRPC Python subcodebase we maintain a no-blank-line-between-behavior-doc-string-and-behavior-body style.

Reply:
ah thanks; will fix